### DISCUSSION TITLE
The msg_id is too low, the client time has to be synchronized Error

### DISCUSSION THREAD

**Discussion by BaZZokaX1** [Dineshkarthik/telegram_media_downloader#448] [Q&A] [answered]
Hi , i was used this script many time about more than one month without any problems but today when i tray to use it it show me this error messgae (pyrogram.errors.BadMsgNotification: [16] The msg_id is too low, the client time has to be synchronized) .. how can i solve it ?

**Comment by dzwzqlx** [comment, marked as answer]
the owner's solution: 
https://github.com/Dineshkarthik/telegram_media_downloader/issues/446#issuecomment-1595644384

**Reply by BaZZokaX1** [reply]
Thanks bro it's work
